docs/src/gettingstarted/introduction/own-code/app-configuration.md

```diff
@@ -104,7 +104,7 @@ There are a few additional keys in `.platform.app.yaml` you will likely need to
 
 * `web`: The `web` key configures the web server through a single web instance container running a single Nginx server process, behind which runs your application.
 
-    * `commands`: Defines the [command](/configuration/app/app-reference.md#commands) to actually launch the application. The `start` key launches your application. In all languages except for PHP, `web.commands.start` should be treated as required. For PHP, you will instead need to define a script name in `passthru`, described below in `locations`.
+    * `commands`: Defines the [command](/configuration/app/app-reference.md#web-commands) to actually launch the application. The `start` key launches your application. In all languages except for PHP, `web.commands.start` should be treated as required. For PHP, you will instead need to define a script name in `passthru`, described below in `locations`.
```

Comment:
```suggestion
    * `commands`: Defines the [command](/configuration/app/app-reference.md#web-commands) to actually launch the application. The `start` key launches your application. In all languages except for PHP, `web.commands.start` should be treated as required. For PHP, you need to define a script name in `passthru` instead, as described below in `locations`.
```

Reply:
I wouldn't put "instead" towards the end of the sentence as it increases the difficulty of parsing it. I'd be hesitant about the "as" too, given the current page structure.

Reply:
Would that be better?

```suggestion
    * `commands`: Defines the [command](/configuration/app/app-reference.md#web-commands) to actually launch the application. The `start` key launches your application. In all languages except for PHP, `web.commands.start` should be treated as required. For PHP, you need to define a script name in `passthru`, described below in `locations`.
```

Reply:
I kept the "instead" in place because even removing the "will" is out of scope for this change.